eng/pipelines/templates/stages/archetype-sdk-tool-azure-function.yml

```diff
@@ -0,0 +1,93 @@
+parameters:
+- name: ToolName
+  type: string
+- name: FunctionProject
+  type: string
+- name: TestProject
+  type: string
+- name: ProductionEnvironmentName
+  type: string
+- name: StagingEnvironmentName
+  type: string
+
+stages:
+    - stage: Build
+      variables:
+      - template: ../variables/globals.yml
+      jobs:
+      - job: BuildPackage
+        pool:
+          vmImage: ubuntu-16.04
+        steps:
+          - task: UseDotNet@2
+            displayName: Tools
+            inputs:
+              version: '3.1.201'
+          - pwsh: |
+              dotnet restore tools/${{ parameters.ToolName }}
```

Comment:
Why separate this from build? build will do it automatically.

Reply:
Will collapse them together.